metrics/utils/ocr_frame_align.sh

```diff
@@ -0,0 +1,537 @@
+#!/bin/bash
```

Comment:
Does this script support parallel executing?

Reply:
I think it not support to do this. Because every step depends on the result of the previous step.

Reply:
I mean that could we use this script to process multiple files at a same time. This script looks like generating some temporary files, is there any conflict for different input?

Reply:
I think the script is ok to do this. By using parameter -p, it can set different prefix of temporary files to avoid name conflict.

Reply:
I didn't see this parameter in your code.

```


<!--StartFragment-->        cmd = [self.file_path, "-p=%s" % (os.path.splitext(os.path.basename(src_video_info.video_path))[0]), "--src=%s" % (src_video_info.video_path), \                "--src_out=%s" % (fp_new_src_video.name), "--fps=%s" % (src_video_info.fps), "-d=%s" % (src_video_info.duration_sec), \                "-w=%s" % (src_video_info.width), "-h=%s" % (src_video_info.height), "--suffix=%s" % (src_video_info.format_abbreviation)]        out = subprocess.run(cmd, check=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, encoding="utf8")<!--EndFragment-->


```